src/pages/DrumMachinePage.js

```diff
@@ -157,10 +155,12 @@ export default function DrumMachinePage({
 
   ////////////////////record////////////////////
   function recordStartClick() {
+    console.log('start rec', recorder);
```

Comment:
maybe this line could deleted to keep the code clean ;) same line 163

Reply:
thx